engine/runtime/src/main/scala/org/enso/interpreter/runtime/DefaultPackageRepository.scala

```diff
@@ -81,10 +82,16 @@ private class DefaultPackageRepository(
     */
   private val loadedPackages
     : collection.mutable.Map[LibraryName, Option[Package[TruffleFile]]] = {
-    val builtinsName = LibraryName(Builtins.NAMESPACE, Builtins.PACKAGE_NAME)
     collection.mutable.LinkedHashMap(builtinsName -> None)
   }
 
+  private def builtinsName: LibraryName = {
```

Comment:
```suggestion
  private val builtinsName: LibraryName = {
```

Reply:
- wouldn't `val` create a variable in the object
- `def` should (imo) only create a function
- e.g. avoid any memory changes/overhead